Sources/KSCrashRecording/KSCrash.m

```diff
@@ -325,14 +249,10 @@ - (NSDictionary*) systemInfo
     return dict;
 }
 
-- (BOOL) install
+- (BOOL) installWithConfiguration:(KSCrashConfiguration*) configuration
 {
-    _monitoring = kscrash_install(self.bundleName.UTF8String,
-                                          self.basePath.UTF8String);
-    if(self.monitoring == 0)
-    {
-        return false;
-    }
+    self.configuration = [configuration copy];
+    kscrash_install(self.bundleName.UTF8String, self.basePath.UTF8String, [self.configuration toCConfiguration]);
```

Comment:
What if `self.configuration == nil`? Maybe it's better to have a function that accepts a potentially nil ObjC config and returns structure (default if nil)

Reply:
I think `self.configuration = [configuration copy] ?: [KSCrashConfiguration new];` would be enough, as `kscrash_install` accepts struct, not a pointer.

Reply:
Yep, that's good too